src/index.html

```diff
@@ -1,17 +1,156 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <div class="inputLine">
+          <label for="surname">Surname: </label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off">
```

Comment:
Closing bracket should be on the same level as the opening one.
```suggestion
          <input
            type="text"
            id="surname"
            name="surname"
            autocomplete="off"
          >
```